packages/api-client/src/main/http/HttpClient.ts

```diff
@@ -129,8 +130,8 @@ class HttpClient extends EventEmitter {
           return Promise.reject(networkError);
         }
 
-        if (isUnauthorized) {
-          return this.refreshAccessToken().then(() => this._sendRequest(config, tokenAsParam));
+        if ((isUnauthorized || isForbidden) && isBackendError && error.response.data.message === 'Token expired') {
```

Comment:
I am concerned about the `Token expired` message... What happens if our backend team changes that message?

Reply:
Understandable. But this is the only difference between an error caused by an expired token and an error with invalid credentials.

Reply:
If you entered invalid credentials, then you don't have a `context` but when entering valid credentials and later on dropping because of an expired access token, then the `context` is set. Can we work with this state?

Reply:
I am now checking if an access token exists.

Reply:
That's great! 😃